RxSwiftWorkshopTests/Utilities/GentleUnwrapping.swift

```diff
@@ -0,0 +1,32 @@
+//
+// Created by Krzysztof Siejkowski on 03/05/2017.
+// Copyright (c) 2017 Mobile Academy. All rights reserved.
+//
+
+import Foundation
+import Nimble
+
+func unwrap<A>(_ a: A?, file: FileString = #file, line: UInt = #line) -> A? {
+    if let _ = a {} else { fail("Object of type \(A.self) is nil", file: file, line: line) }
```

Comment:
Guard enforces return and therefore if there're more objects that are surprisingly nil, you'll never know. The reason for `unwrap` is to provide a nice test result logs with the information of what went wrong, so I'd like to avoid striping the information as much as possible.